models/intermediate/hubspot/int_rag_hubspot__deal_comment_document.sql

```diff
@@ -126,7 +126,7 @@ comment_markdowns as (
         comment_time,
```

Comment:
Could you add `title` to
* here
* the second query you're unioning
* the following CTEs
* the `int_rag_hubspot__deal_comment_documents_grouped` model

to persist it downstream